intellst-api/src/Service/Register.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace App\Service;
+
+use App\Entity\User;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\RequestStack;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Security\Core\Encoder\UserPasswordEncoderInterface;
+use Doctrine\ORM\getDoctrine;
+
+class Register extends AbstractController
```

Comment:
why is it controller??

Reply:
plese can we chenge that o the other branch registration?